src/client/components/SlateEditor/SlateContent.less

```diff
@@ -8,6 +8,7 @@
   position: relative;
   font-size: 14px;
   font-family: Ubuntu Mono, Monaco, monospace;
+  padding-left: 7ch;
```

Comment:
`H6` looks too close to left margin of editor. Maybe increase left padding `7ch` => `8ch` ?